apps/src/sites/studio/pages/code-studio.js

```diff
@@ -46,6 +46,7 @@ window.dashboard.assets = require('@cdo/apps/code-studio/assets');
 window.dashboard.pairing = require('@cdo/apps/code-studio/pairing');
 window.dashboard.teacher = require('@cdo/apps/code-studio/teacher');
 window.dashboard.project = require('@cdo/apps/code-studio/initApp/project');
+window.dashboard.schoolInfoBox = require('@cdo/apps/code-studio/districtDropdown/SchoolInfoBox.jsx');
```

Comment:
we'd prefer not to add things to this namespace if we can avoid it. An example of how we accomplish that : https://github.com/code-dot-org/code-dot-org/pull/11322